deploy/docker-compose.yml

```diff
@@ -0,0 +1,123 @@
+version: "3"
+
+networks:
+  frontend:
+  backend:
+
+volumes:
+  private-ethereum-blockchain:
+
+services:
+
+  supplyChainServer:
+    image: 235945186688.dkr.ecr.ap-southeast-2.amazonaws.com/supplychainserver:latest
```

Comment:
Given this is an open-source project, should actually be using a private docker image repo?

Reply:
My gut says Yes!

I've looked into it and have not found any ways I'm keen on to authenticate with github.com so that we can push images though. All paths appear to need a username/password for a `docker login`, or an auth token which expires after time and so would need to be updated regularly, as well as still being tied to a specific user.